use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,87 +3,53 @@ design_pattern_id: 2
 time_in_minutes: 60
 difficulty_level: Intermediate
 ---
 
-### Presenting the easiest way for your employees to Take Time Off 🏝️
+### Take Time Off with your Copilot 🏝️
 
-Employees often need to take unplanned or planned time off, making it important for managers to review time off requests ahead of time.
+Your employee deserve to take time off to recharge and be productive, and time off request surfaces in Workday don't make this process easy.
```

Comment:
Questionable if we're attacking Workday -- especially since we want to do partnerships. If we're going to call this out, might be worth reviewing lines like these with Dave to avoid creating bad blood

Reply:
updated